docs/content/en/docs/overview/partner/_index.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Partners"
+linkTitle: "Partners"
+weight: 30
+description: >
+  EKS Anywhere validated partners
+---
+
+Amazon EKS Anywhere maintains relationships with third-party vendors to provide add-on solutions for EKS Anywhere clusters.
+A complete list of these partners is maintained on the [Amazon EKS Anywhere Partners](https://aws.amazon.com/eks/eks-anywhere/partners/) page.
+The following shows validated EKS Anywhere partners whose products have passed conformance test for specific EKS Anywhere providers and versions:
+
```

Comment:
@elamaran11 do we have the version of the partner software that was validated? Would be good to add if so

Reply:
@csplinter Partner software is not reported in configmap to read to reporting though Partner software has version in helm charts. We have to do some work from our end to add configmap to all partner softwares where we can maintain version for reporting. I will work on this in my backlog to report to this report in future.

Reply:
Okay. Going ahead with the PR, but we'll add that information when it is available.